SettingsList.py

```diff
@@ -3477,6 +3485,11 @@ def __init__(self, name, gui_text, min, max, default, step=1,
             will add 2 more possible locations to each Dungeons.
             This makes dungeons more profitable, especially
             Ice Cavern, Water Temple, and Jabu Jabu's Belly.
+            
+            Regardless of the selected option, maps and compasses from
```

Comment:
I think the "Empty Dungeons" setting should be mentioned explicitly to avoid confusion with randomly rolled barren dungeons. Also for the (boss) keysanity tooltips.

Reply:
What about starting the paragraph with `If "Empty Dungeons" is on, then regardless of the selected option...`?
Or that also raise a more general question: maybe _Empty dungeons_ is not clear enough and we can find a better wording that leaves no ambiguity with 'naturally' barren dungeons? When @ArthurOudini first presented me this concept, he referred to them as _Banned dungeons_ (having in mind some pick and ban system for 1v1 races for example). That could be an option too.

Reply:
I agree that it would probably be best to come up with a phrasing that is less ambiguous.

Reply:
"Vacant" and "Useless" are also being suggested.

Reply:
Following discussions on Discord and some more brainstorming, maybe we can narrow down to the names below:
 - vacant dungeon,
 - removed dungeon,
 - cancelled dungeon,
 - nullified dungeon.
 
 Being not fluent in English, I don't know which one describes best _a dungeon filled with junk **and** whose dungeon reward is given for free at the very beginning of the seed_, while not adding confusion with randomly rolled _barren dungeons_. Maybe we can just make a poll on OoTR Discord but I don't know if this is a common way to proceed and I don't want to spam #dev-public-talk.

Once a definitive name is fixed I will commit a tooltip update taking in account all above comments.